src/Workspaces/Core/Portable/Storage/SQLite/SQLitePersistentStorage.Accessor.cs

```diff
@@ -24,28 +21,21 @@ internal partial class SQLitePersistentStorage
         private abstract class Accessor<TKey, TWriteQueueKey, TDatabaseId>
         {
             protected readonly SQLitePersistentStorage Storage;
-            private readonly string _select_rowid_from_0_where_1;
-            private readonly string _insert_or_replace_into_0_1_2_3_value;
-
-            /// <summary>
-            /// Queue of actions we want to perform all at once against the DB in a single transaction.
-            /// </summary>
-            private readonly MultiDictionary<TWriteQueueKey, Action<SqlConnection>> _writeQueueKeyToWrites =
-                new MultiDictionary<TWriteQueueKey, Action<SqlConnection>>();
-
-            /// <summary>
-            /// The task responsible for writing out all the batched actions we have for a particular
-            /// queue.  When new reads come in for that queue they can 'await' this write-task completing
-            /// so that all reads for the queue observe any previously completed writes.
-            /// </summary>
-            private readonly Dictionary<TWriteQueueKey, Task> _writeQueueKeyToWriteTask =
-                new Dictionary<TWriteQueueKey, Task>();
+
+            private readonly string _select_rowid_from_main_0_where_1;
+            private readonly string _select_rowid_from_writecache_0_where_1;
+            private readonly string _insert_or_replace_into_writecache_0_1_2_3_value;
+            private readonly string _delete_from_writecache_0;
+            private readonly string _insert_or_replace_into_main_0_select_star_from_writecache_1;
```

Comment:
nearly the same queries as before.  Except we cache the strings for the main/writecache db forms to prevent continually allocating them.